gen_manifest/gen_manifest.py

```diff
@@ -251,9 +248,13 @@ def get_region_tag(sample_file_path):
   return region_tags[0]
 
 
-def glob_non_yaml(pattern):
-  '''Recursively globs for `pattern`, ignoring "*.yaml" and ".yml" files.'''
-  matches = glob(pattern, recursive=True)
+def glob_non_yaml(all_patterns):
+  '''Recursively globs for each pattern in `all_patterns`, ignoring "*.yaml" and ".yml" files.'''
+  matches = []
+  for pattern in all_patterns:
+      matches.extend(glob(pattern, recursive=True))
+  matches = list(set(matches)) # eliminate duplicates
+  matches.sort()               # want deterministic order for tests
   return [root+ext
           for root, ext in [os.path.splitext(path) for path in matches]
           if ext != ".yaml" and ext !=".yml"]
```

Comment:
While we're changing things, this could be made a little simpler like so:
```python
yaml_exts = (".yaml", ".yml")
return [path for path in matches if os.path.splitext(path)[1] not in yaml_exts]
```